src/postamble.js

```diff
@@ -139,7 +139,7 @@ function run() {
 
 #if PTHREADS || WASM_WORKERS
   if ({{{ ENVIRONMENT_IS_WORKER_THREAD() }}}) {
-#if MODULARIZE
+#if USE_READY_PROMISE
     readyPromiseResolve?.(Module);
```

Comment:
I think you can perhaps remove the question marks here too. i.e. we can make USE_READY_PROMISE imply that readyPromiseResolve exists?